apps/meteor/client/components/AutoCompleteMultiple/AutoCompleteMultiple.tsx

```diff
@@ -0,0 +1,88 @@
+import { AutoComplete, Box, Option, OptionAvatar, OptionContent, Chip } from '@rocket.chat/fuselage';
+import { useMutableCallback, useDebouncedValue } from '@rocket.chat/fuselage-hooks';
+import React, { ComponentProps, memo, ReactElement, useMemo, useState } from 'react';
+
+import { useEndpointData } from '../../hooks/useEndpointData';
+import UserAvatar from '../avatar/UserAvatar';
+
+const query = (
+	term = '',
+): {
+	selector: string;
+} => ({ selector: JSON.stringify({ term }) });
+type roomType = {
+	label: string;
+	value: string;
+	_id: string;
+	type: string;
+};
+type AutoCompleteMultipleProps = Omit<ComponentProps<typeof AutoComplete>, 'value' | 'filter' | 'onChange'> &
+	Omit<ComponentProps<typeof Option>, 'value' | 'is' | 'className' | 'onChange'> & {
+		onChange: (room: roomType, action: 'remove' | undefined) => void;
+		value: any;
+		filter?: string;
+	};
+
+const AutoCompleteMultiple = ({ onChange, ...props }: AutoCompleteMultipleProps): ReactElement => {
```

Comment:
Let's use a different name for this one, I suggest something more descriptive like `UsersAndRoomAutoCompleteMultiple`